### ISSUE TITLE
PanelBar [selectable]="false" Prevents Expand/Collapse

### ISSUE THREAD

**Issue by barnesew** [telerik/kendo-angular#3676]
## Describe the bug

When using the Kendo PanelBar, if you set the selectable input to false, it prevents a user from clicking any of the PanelBarInputs to expand/collapse them.

The desired behavior was observed to be working in `@progress/kendo-angular-layout@6.5.2` but observed to be broken in `@progress/kendo-angular-layout@6.6.0`.

## To Reproduce

Here is a StackBlitz that was forked from the PanelBar example that shows the issue:
https://stackblitz.com/edit/kendo-panelbar-selectable-issue

I was unable to get a second stackblitz working due to dependency issues, but here is an example project where the PanelBar functionality was working:
[kendo-panelbar-selectable-working.zip](https://github.com/telerik/kendo-angular/files/8730788/kendo-panelbar-selectable-working.zip)

## Expected behavior

I would expect that even if a PanelBarItem cannot be selected, it should still be able to be expanded/collapsed.

## Browser
 - OS: macOS Monterey (12.2.1)
 - Browser: Chrome
 - Version: 101.0.4951.64 (arm64)

### Working Example Packages:
```bash
example-app@1.0.0 /Users/barnesew/Downloads/kendo-panelbar-selectable-issue
├── __ngcc_entry_points__.json@ extraneous
├── @angular-devkit/build-angular@12.2.17
├── @angular/animations@12.2.16
├── @angular/cli@12.2.17
├── @angular/common@12.2.16
├── @angular/compiler-cli@12.2.16
├── @angular/compiler@12.2.16
├── @angular/core@12.2.16
├── @angular/forms@12.2.16
├── @angular/localize@12.2.16
├── @angular/platform-browser-dynamic@12.2.16
├── @angular/platform-browser@12.2.16
├── @angular/router@12.2.16
├── @progress/kendo-angular-buttons@7.0.6
├── @progress/kendo-angular-common@2.0.3
├── @progress/kendo-angular-dropdowns@6.0.2
├── @progress/kendo-angular-icons@1.0.1
├── @progress/kendo-angular-inputs@8.0.9
├── @progress/kendo-angular-intl@3.1.3
├── @progress/kendo-angular-l10n@3.0.4
├── @progress/kendo-angular-label@3.1.3
├── @progress/kendo-angular-layout@6.5.2
├── @progress/kendo-angular-popup@4.0.6
├── @progress/kendo-angular-progressbar@2.0.4
├── @progress/kendo-angular-treeview@6.0.2
├── @progress/kendo-drawing@1.16.3
├── @progress/kendo-licensing@1.2.2
├── @progress/kendo-svg-icons@0.1.2
├── @types/node@12.11.1
├── core-js@3.21.1
├── rxjs@7.5.2
├── tslib@2.3.0
├── typescript@4.3.4
├── web-animations-js@2.3.2
└── zone.js@0.11.5
```

**Comment by Raisolution** [comment]
Fix available in `v7.0.2` of the Layout package.

**Comment by barnesew** [comment]
Thanks for the quick turnaround!
